gramps/gen/filters/rules/family/_hastwins.py

```diff
@@ -48,18 +57,19 @@ class HasTwins(Rule):
     description = _("Matches families with twins")
     category = _("Child filters")
 
-    def apply(self, db, family):
+    def apply_to_one(self, db: Database, family: Family) -> bool:
         date_list = []
-        for childref in family.get_child_ref_list():
-            if int(childref.get_mother_relation()) == ChildRefType.BIRTH:
+        for childref in family.child_ref_list:
+            if int(childref.mrel) == ChildRefType.BIRTH:
                 child = db.get_person_from_handle(childref.ref)
-                birthref = child.get_birth_ref()
-                if birthref:
-                    birth = db.get_event_from_handle(birthref.ref)
-                    sortval = birth.get_date_object().get_sort_value()
-                    if sortval != 0:
-                        if sortval in date_list:
-                            return True
-                        else:
-                            date_list.append(sortval)
+                if 0 <= child.birth_ref_index < len(child.event_ref_list):
+                    birthref = child.event_ref_list[child.birth_ref_index]
+                    if birthref:
+                        birth = db.get_event_from_handle(birthref.ref)
+                        sortval = birth.date.sortval
+                        if sortval != 0:
+                            if sortval in date_list:
+                                return True
+                            else:
+                                date_list.append(sortval)
```

Comment:
```suggestion
                            date_list.append(sortval)
```
Unnecessary else.

Reply:
I think it is more explicit, and "Explicit is better than implicit."